recipes/pennies/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "pennies" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5044bfd6009dc4d6d14ed8d8355d0cc4b9cb4aafad105b80b0c481f8beacdc15
+  patches:
+    - 0001-fix-imports.patch
```

Comment:
Does this need to be patched? Unless there is a strong argument to do it, I'd fix it in the next release upstream.

Reply:
Hi. Wow - I had somehow missed all of this conversation thread! Thanks for all your work @synapticarbors! The bug was actually created specifically so that I would have to make a patch in the recipe. I did this for a blog post: https://www.anaconda.com/patching-source-code-to-conda-build-recipes/

If Anaconda, at which I no longer work, lets me update that post, then yes, I will certainly clean up this recipe, too. I now no longer remember whether I created the recipe entirely for that post. At the time, I was creating a ton of recipes for an Enterprise client. Anyway, having the experience of maintaining my first recipe  on conda-forge is important to me, so I'll pick this back up before github's stale-bot eats my homework.